Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h

```diff
@@ -53,15 +53,17 @@ class DetGeomDesc {
   using RotationMatrix = ROOT::Math::Rotation3D;
   using Translation = ROOT::Math::DisplacementVector3D<ROOT::Math::Cartesian3D<double>>;
 
+  DetGeomDesc() = default;
```

Comment:
Why adding the default constructor?
A DetGeomDesc built with this, will not have interesting data members, and no way of setting them.

Reply:
A way of creating an "empty" object is needed here:
  https://github.com/cms-sw/cmssw/pull/31492/files/0e26c4a0f7658a37176349a8e446492503cdf81c#diff-cae30eec6c3de96aa4e668edf94cf7a8R62
Would you have a better suggestion?

Reply:
Make it private?

Reply:
I think there should just be a custom copy constructor. 
This in a sense is just 'trying to have' a constructor, with settings done in makeCopyWithoutChildren()